crates/biome_cli/tests/snapshots/main_cases_biome_json_support/biome_json_is_not_ignored.snap

```diff
@@ -28,7 +28,9 @@ expression: content
 # Emitted Messages
 
 ```block
-Checked 2 files in <TIME>. No fixes needed.
+Checked 2 files in <TIME>.
 ```
 
-
+```block
```

Comment:
I'm not really a fan of splitting this into separate blocks. They'll be separated in the console output too, right? I would like to search for a way to keep the old "more opinionated" output format, since it feels quite verbose this way.